gui.py

```diff
@@ -0,0 +1,82 @@
+from conversion_manager import ConversionManager
+from path_helper import PathHelper
+from PyQt6.QtCore import QSize
+from PyQt6.QtGui import QIcon
+from PyQt6.QtWidgets import QApplication, QMainWindow, QPushButton, QVBoxLayout, QHBoxLayout, QLabel, \
+    QWidget, QFileDialog, QMessageBox
+
+
+class MainWindow(QMainWindow):
+    def __init__(self, conversion_manager: ConversionManager, input_path: str, output_path: str) -> None:
+        super().__init__()
+        self.setMinimumSize(QSize(300, 150))
+
+        self.conversion_manager: ConversionManager = conversion_manager
+        self.input_path: str = input_path
+        self.output_path: str = output_path
+
+        self.setWindowTitle("Аршин-конвертер")
+        self.layout = QVBoxLayout()
+        button_layout = QHBoxLayout()
+
+        search_button = QPushButton("Обзор")
+        start_button = QPushButton("Начать")
+        search_button.clicked.connect(self.__the_search_button_was_clicked)
+        start_button.clicked.connect(self.__the_start_button_was_clicked)
+
+        if self.input_path:
+            self.input_path = PathHelper.get_abspath(self.input_path)
+            text_input_path_label = 'Выбран файл: ' + self.input_path
+        else:
+            text_input_path_label = 'Выберите файл для конвертации'
+        self.input_path_label = QLabel(text_input_path_label)
+
+        text_output_path_label = 'Сохранить в: '
+        if self.output_path:
+            self.output_path = PathHelper.get_abspath(self.output_path)
+            text_output_path_label += self.output_path
+        self.output_path_label = QLabel(text_output_path_label)
+
+        button_layout.addWidget(search_button)
+        button_layout.addWidget(start_button)
+
+        self.layout.addWidget(self.input_path_label)
+        self.layout.addWidget(self.output_path_label)
+        self.layout.addLayout(button_layout)
+
+        widget = QWidget()
+        widget.setLayout(self.layout)
+        self.setCentralWidget(widget)
+
+    def __the_search_button_was_clicked(self) -> None:
```

Comment:
Although definite article is a correct option here, it's a common practice to avoid it in the naming.